tests/scripts/combined-coverage.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 
 npm run test:unit
+CURRENT_NODE_VERSION=`node -v`
+if [[ $CURRENT_NODE_VERSION =~ 17(.*)|18(.*) ]]
+then
+  echo -e "currently using node version 17+ - setting NODE_OPTIONS \n"
+  export NODE_OPTIONS=--openssl-legacy-provider
```

Comment:
can we do this within the github workflow instead? https://github.com/yext/search-ui-react/blob/dev/v18-visual-test/.github/workflows/run-tests.yml

Reply:
Ideally in the workflows we would only set this in the node 18 jobs just like you have here. We would probably need to separate out 18.x from the specified node matrices

Reply:
Should I still keep the changes in the test script? I was thinking keeping them would allow visual test to run locally with node 17/18

Reply:
oh I see, that's fair!

Reply:
Could you go back to what you had before, but instead of putting it in combined-coverage.sh make a new script for getting the visual coverage? otherwise we won't be able to run `npm run test:visual` on its own

a separate question, if you export NODE_OPTIONS in this script, does it persist outside of the script? if it does, I think we want to specify it inline, something like `NODE_OPTIONS=blah npx jest ... `

Reply:
I tested it in node 16 with a test script to set option since the test would error out of NODE_OPTION is set to --openssl-legacy-provider. But after running set NODE_OPTION bash script, the test still runs. So I think setting the NODE_OPTION only works within the script. I added an unset NODE_OPTION at the end of the visual test script just to be safe but it's most likely not necessary.